src/main/java/org/reaktivity/nukleus/http2/internal/Http2Connection.java

```diff
@@ -1286,9 +1286,9 @@ private void mapToHttp(DirectBuffer name, DirectBuffer value)
     {
         if (!headersContext.error())
         {
-            factory.httpBeginExRW.headers(b -> b.item(item -> item.representation((byte) 0)
+            factory.httpBeginExRW.headersItem(item -> item.representation((byte) 0)
```

Comment:
Please remove `item.representation((byte) 0)` call as it is already defaulted to `0`.

Reply:
done